app/views/shared/_conversation_sidebar_link.html.erb

```diff
@@ -1,20 +1,21 @@
 <%# locals: (conversation:, settings: true) %>
 <% selected = conversation == @conversation %>
 
-<div class="flex rounded-lg hover:bg-zinc-800 <%= selected && 'bg-zinc-800' %> justify-between items-center group">
-  <%= link_to conversation.title, conversation, class: "flex-1 pl-2 py-2 text-white truncate text-sm text-opacity-80 w-5" %>
+<div class="flex rounded-lg hover:bg-zinc-800 <%= selected && 'bg-zinc-800' %> justify-between items-center group mr-5 cursor-pointer">
```

Comment:
Adding cursor-pointer to this line is what fixed the javascript error. We want it on this overall element because we want it to look like it's depressing upon click (that's a styled behavior I added to the cursor-pointer class).

Reply:
I'm not sure I understand all of what the nested-pointer stuff is doing yet, but that's ok. 👍